configs/config

```diff
@@ -1,7 +1,6 @@
 BR2_arm=y
-BR2_arm1176jzf_s=y
 BR2_TOOLCHAIN_BUILDROOT_GLIBC=y
-BR2_KERNEL_HEADERS_5_4=y
+BR2_PACKAGE_HOST_LINUX_HEADERS_CUSTOM_5_9=y
```

Comment:
Why not `BR2_KERNEL_HEADERS_5_9`?

Reply:
This is actually my first time touching buildroot, `KERNEL_HEADERS` is headers used for compiling for device, and `PACKAGE_HOST_LINUX_HEADERS` are the headers used for compiling host tools?

I'll change to your recommendation, that seems to actually make sense :sweat_smile: